PurchasesTests/FoundationExtensions/DictionaryExtensionsTests.swift

```diff
@@ -87,4 +87,15 @@ class DictionaryExtensionsTests: XCTestCase {
         expect(obtainedDict).to(equal(expectedDict))
     }
 
+    func testAddEntriesToDictionaryWithOperatorPlusAddsValuesCorrectly() {
```

Comment:
Thanks for writing tests for this btw :+1:

Reply:
Thanks but I can't take credit, it's a small extension on @Juanpe's contributions here https://github.com/RevenueCat/purchases-ios/pull/716/